src/App.js

```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
```

Comment:
Why are you importing `MyComponent` if you are not using it?

Reply:
It was for testing I'm going to use it later but it's true for the moment it's not useful.